engine/routines.go

```diff
@@ -336,8 +342,17 @@ func (bot *Engine) WebsocketDataHandler(exchName string, data interface{}) error
 			od.UpdateOrderFromDetail(d)
 		}
 	case *order.Cancel:
+		if bot.Settings.Verbose {
+			printOrderCancelSummary(d)
+		}
+		// TODO: This can be inferred by its status so no exceess type needs
```

Comment:
Yeah, that might be better. I just didn't want to change functionality too much with this pass. And keep websocket kind of the same while outputting the correct information. As with the cancelled, in this context the matching engine can cancel an order, you will have a pending come through and if you have a time in force (FOK) it will auto cancel on you. This case won't be hit on this update but I had it there as I check and update the other implementations.

Reply:
Orders that have had their status been changed server side (as in, GCT has not made the request to cancel the order) should go through order.modify since order.Cancel cancels the order via API (and also has a log that says its cancelling it). The cancel summary being before with the words "Order Cancelled" is misleading and happens before any error processing has occurred.

Reply:
Thanks for clarifying that, Yeah, I just deleted it. There was only one instance in Bitmex where we were getting that case. Must have been added by accident. 🤷